assets/quick-add.css

```diff
@@ -0,0 +1,196 @@
+.quick-add {
+  position: relative;
+  grid-row-start: 4;
+  margin: 0 0 1rem;
+  z-index: 1;
+}
+
+.card--card .quick-add {
+  margin: 0 1.3rem 1rem;
+}
+
+.quick-add-modal {
+  box-sizing: border-box;
+  opacity: 0;
+  position: fixed;
+  visibility: hidden;
+  z-index: -1;
+  margin: 0 auto;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background: rgba(var(--color-foreground), 0.2);
+  height: 100%;
+}
+
+.quick-add-modal[open] {
+  opacity: 1;
+  visibility: visible;
+  z-index: 101;
+}
+
+.quick-add-modal__content {
+  position: absolute;
+  top: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  margin: 3.2rem auto 0;
+  width: 100%;
+  background-color: rgb(var(--color-background));
+  overflow: hidden;
+  max-width: var(--page-width);
+  width: calc(100% - 3rem);
+}
+
+@media screen and (min-width: 750px) {
+  .quick-add-modal__content {
+    margin-top: 10rem;
+    width: 80%;
+    height: calc(100% - 20rem);
+    overflow-y: auto;
+  }
+
+  quick-add-modal .quick-add-modal__toggle {
+    top: 2rem;
+    right: 2rem;
+  }
+}
+
+@media screen and (min-width: 990px) {
+  .quick-add-modal__content {
+    width: 70%;
+  }
+}
+
+.quick-add-modal__content img {
+  max-width: 100%;
+}
+
+.quick-add-modal__content-info {
+  padding-right: 4.4rem;
+  display: flex;
+  overflow-y: auto;
+  padding: 2.5rem;
```

Comment:
I think adding this padding here is preventing the product` ul.slider` element within from stretching the fullwidth. If the slider was being used as one, this would be obvious when you scrolled to the next image. The slider of course is still active as well https://screenshot.click/24-45-tju54-wfqcs.mp4. This is because we don't conditionally remove the slider classes or anything when the slider has only 1 _visible_ item. I'd advise we don't tackle that existing problem in this PR but the video helps show what the issue with the padding is.

<img src="https://screenshot.click/24-41-q0kwq-c88uk.png" width="300" />

Reply:
If the quick add modal is only showing 1 media no matter what right now though we can potentially just fully override the slider overflow within quick add for now.